src/Microsoft.ML.Data/DataLoadSave/DataOperationsCatalog.cs

```diff
@@ -90,5 +90,27 @@ public IDataView FilterByKeyColumnFraction(IDataView input, string columnName, d
                 throw Environment.ExceptSchemaMismatch(nameof(columnName), "filter", columnName, "KeyType", type.ToString());
             return new RangeFilter(Environment, input, columnName, lowerBound, upperBound, false);
         }
+
+        /// <summary>
+        /// Drop rows where any column in <paramref name="columns"/> contains a missing value.
+        /// </summary>
+        /// <param name="input">The input data.</param>
+        /// <param name="complement">If true, keep only rows that contain missing values, and filter the rest.</param>
+        /// <param name="columns">Name of the columns to filter on. If a row is has a missing value in any of
+        /// these columns, it will be dropped from the dataset.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[MissingValueFilter](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/MissingValueFilter.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
+        public IDataView MissingValueFilter(IDataView input, bool complement = false, params string[] columns)
```

Comment:
>MissingValueFilter [](start = 25, length = 18)

Is the name compliant with our naming guideline? Shouldn't it be FilterMissingValues? Or FilterByMissingValues to match FilterByColumn and FilterByKeyColumnFraction above?

Reply:
You are right. Made it declarative.

---
In reply to: [254077054](https://github.com/dotnet/machinelearning/pull/2424#discussion_r254077054) [](ancestors = 254077054)